cogeo_mosaic/backends/dynamodb.py

```diff
@@ -124,18 +125,19 @@ def read(self) -> MosaicJSON:
         return MosaicJSON(**meta)
 
     @functools.lru_cache(maxsize=512)
-    def get_assets(self, x: int, y: int, z: int) -> Tuple[str]:
+    def get_assets(self, x: int, y: int, z: int) -> Sequence[str]:
```

Comment:
Why `Sequence` instead of `Tuple`?

Reply:
Because we are returning a `list`. I could put `List` but I think Sequence is usually better